pages/create-account.js

```diff
@@ -33,13 +37,18 @@ export default function CreateAccount() {
           errorMessage = input.length === 0 ?
             'Please provide a last name!' : '';
           break;
+        case 'email':
+          errorMessage = !isEmail(input) ?
+            'Please provide a valid email!' : '';
+          break;
         case 'username':
           errorMessage = input.length < 5 ? 
             'Username must be at least 5 characters long!' : '';
           break;
         case 'password':
-          errorMessage = input.length < 5 ?
-            'Password must be at least 6 characters long!' : '';
+          const passwordRegex = /^(?=.*[0-9!@#$%^&*])[A-Za-z0-9!@#$%^&*]{6,}$/;
```

Comment:
This works, but is there a library that can test the password for us? If not, add a comment describing what the regex is checking for